src/app/main.cpp

```diff
@@ -302,6 +352,38 @@ QBtCommandLineParameters parseCommandLine()
                 result.shouldDaemonize = true;
             }
 #endif
+            else if ((arg == QLatin1String("-p")) || (arg == QLatin1String("--path"))) {
+                if ((i+1) < appArguments.size()) {
```

Comment:
Here and in similar places you break coding style rule. You must insert spaces around arithmetic operators.